src/neptune/internal/backends/hosted_neptune_backend.py

```diff
@@ -1065,3 +1094,23 @@ def get_model_version_url(
     ) -> str:
         base_url = self.get_display_address()
         return f"{base_url}/{workspace}/{project_name}/m/{model_id}/v/{sys_id}"
+
+
+def _get_column_type_from_entries(entries: List[Any], column: str) -> str:
+    if not entries:  # column chosen is not present in the table - no sorting will be done anyway
+        return AttributeType.STRING.value
+
```

Comment:
Please filter the entries by an exact match of a column name (requested vs the one from entries) as to my knowledge this works like a regex `name*`.